config/software/perl.rb

```diff
@@ -35,8 +35,13 @@
   env = with_standard_compiler_flags(with_embedded_path)
 
   solaris_mapfile_path = File.expand_path(Omnibus::Config.solaris_linker_mapfile, Omnibus::Config.project_root)
-  if solaris2? && File.exist?(solaris_mapfile_path)
-    cc_command = "-Dcc='gcc -static-libgcc -Wl,-M #{solaris_mapfile_path}"
+  if solaris_10?
+    cc_command = "-Dcc='gcc -static-libgcc'"
+    if File.exist?(solaris_mapfile_path)
```

Comment:
There are a lot of software which requires perl as they contain perl scripts which are used as part of the build. @scotthain could likely answer to this definitively, but I suspect that is the reason we need it.